### ISSUE TITLE
Missing LIFEAlertAction.h file after iOS installation

### ISSUE THREAD

**Issue by ArnaudRinquin** [Buglife/react-native-buglife#7]
Title says it all:

![screen shot 2018-01-24 at 23 11 15](https://user-images.githubusercontent.com/228886/35375463-f7cf0768-015b-11e8-90ac-c8d1070bc0ab.png)

**Comment by schukin** [comment]
Thanks @ArnaudRinquin, this was definitely a build breakage on our part.

We pushed out react-native-buglife v0.1.12 which resolves this. We've verified that it works in a new iOS app, as well as an existing app that is migrated from an older version of react-native-buglife. Please update to the latest react-native-buglife (no code changes should be necessary, even imports).

FWIW, this appears to have been caused by some inconsistencies in how native modules' iOS targets are configured for React Native projects. We've now resolved this in RNBuglife.xcodeproj. _However_, through this we uncovered some issues in React Native itself, which seem to have cropped up recently according to some recent Github issues:

https://github.com/facebook/react-native/issues/17148

We're keeping a close eye on React Native issues involving `RCTEventEmitter`; While this isn't directly related to `react-native-buglife`, it is something that might crop up w/ use of any native modules (Crashlytics, etc) while compiling for iOS.

**tldr; update react-native-buglife to v0.1.12 and things should be back to normal.** Thanks again for your help! 🙌
